symengine/catch/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(catch)
+
+set(SRC
+    catch.cpp
+)
+
+add_library(catch STATIC ${SRC})
+target_link_libraries(catch symengine)
+
+install(TARGETS catch
+            RUNTIME DESTINATION bin
+            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib
+        )
```

Comment:
Cool, I'll remove the install lines.
We can change teuchos header includes later.